src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
+@import "./font-family.scss";
+@import "./navigation.scss";
+@import "./card.scss";
+@import "./reset.scss";
+@import "./stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
move in the tml selector, as Alisa mentioned

https://github.com/mate-academy/layout_catalog/pull/2448#discussion_r1051331640